src/game-container/MainContainer.svelte

```diff
@@ -25,7 +23,9 @@
 </style>
 
 <div class="main-container">
-    <GfxContainer />
+    <GfxContainer
+        {classList}
+     />
     <TxtContainer />
-    <UIContainer />
+    <UIContainer  />
```

Comment:
van 1 naar 2 spaties? hippe shit

Reply:
Btw als alle dingen waarvan ik net vroeg goh zijn dat svelte dingen echt svelte dingen zijn en dit hierboven ook, dan is Svelte echt vrijwel identiek aan vue

Reply:
Ja ze lijken veel op elkaar idd. Alleen is Svelte net wat anders, in de zin dat het in principe een compiler is. Alle .svelte code wordt gecompiled in bundle.js en dat is wat de browser runt